Assets/MixedRealityToolkit.Tests/EditModeTests/Examples/Demos/EyeTracking/DemoVisualizer/Scripts/UserInputRecorderTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using NUnit.Framework;
+using System.Globalization;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos.EyeTracking.Logging
+{
+    class UserInputRecorderTests
+    {
+        [Test]
+        public void TestGetStringFormat()
+        {
+            // This test array is passed into UserInputRecorder.GetStringFormat and
+            // its content doesn't matter, only its length. The length is arbitrarily
+            // chosen to be relatively short.
+            object[] testArray = new object[5];
+
+            // en-US uses comma separated delimiters.
+            CultureInfo.CurrentCulture = new CultureInfo("en-US", false);
+
+            // In en-US, the generated format should be comma-delimited.
+            Assert.AreEqual(
+                "{0}, {1}, {2}, {3}, {4}",
+                UserInputRecorder.GetStringFormat(testArray));
+
+            // de-DE uses semicolon separated delimiters.
+            CultureInfo.CurrentCulture = new CultureInfo("de-DE", false);
+
+            // In de-DE, the generated format should be semicolon-delimited.
+            Assert.AreEqual(
```

Comment:
In addition to checking for the correct delimiter, did you check that eye tracking playback from the original issue works with this new format?

Reply:
I did not!